types/architect__functions/ws.d.ts

```diff
@@ -0,0 +1,22 @@
+import { HttpResponse } from './http';
+
+// https://arc.codes/docs/en/reference/app.arc/ws (current docs site, lacks detail)
+// https://v6.arc.codes/primitives/ws (previous docs site, more info)
+export interface WebsocketEvent {
+    requestContext: {
+        connectionId: string;
+    };
+    isBase64Encoded: boolean;
+    body: string;
+}
+
+// WebSockets still respond with HTTP
+// (usually just a 200 acknowledging the socket is connected)
+export interface WebSocketHandler {
+    (req: WebsocketEvent): Promise<HttpResponse | undefined>;
```

Comment:
Is it a full `HttpResponse` here? I actually don't know, but if it is a subset of `HttpResponse` properties I'd suggest a more specific type.